libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -435,6 +445,24 @@ dictionary ReverseSwapFeesRequest {
     u32? claim_tx_feerate = null;
 };
 
+dictionary FetchOnchainLimitsResponse {
+    u64 min_sat;
+    u64 max_sat;
+    string fees_hash;
+};
+
+enum SwapAmountType {
+    "Send",
+    "Receive",
+};
+
+dictionary PrepareOnchainPaymentRequest {
+    u64 amount_sat;
+    SwapAmountType amount_type;
+    u32 claim_tx_feerate;
+    string fees_hash;
```

Comment:
Same here, I think we can avoid passing the fee hash here. After all the prepareOnchain use the swapper API to retrieve a new fees_hash.

Reply:
Yes, if `fees_hash` is removed from the `onchain_payment_limits` result, this will go as well. Depends on decision above.

Reply:
Done: b6fb822f8f735c74136b767219d7b825dae6cd30